src/Microsoft.AspNet.Mvc.ViewFeatures/ViewComponents/ViewViewComponentResult.cs

```diff
@@ -113,5 +116,8 @@ private static IViewEngine ResolveViewEngine(ViewComponentContext context)
         {
             return context.ViewContext.HttpContext.RequestServices.GetRequiredService<ICompositeViewEngine>();
         }
+
+        private static bool IsApplicationRelativePath(string path) =>
+            path[0] == '~' || path[0] == '/';
```

Comment:
12 PranavPoints awarded. Seriously this is harder to read than the normal function syntax. Please stop, we're worried about you :crying_cat_face: